Makefile

```
@@ -49,3 +66,14 @@ smoke:
 
 simulate:
 	go run ./cmd/simulator -url http://localhost:8080 -vehicles 5 -interval 3s -duration 30s
+
+css: $(TAILWIND_BIN)
+	$(TAILWIND_BIN) -i web/styles/input.css -o web/static/css/admin.css --minify
+
+$(TAILWIND_BIN):
```

checkmake (0.3.2)

[warning] Target body for "$(TAILWIND_BIN)" exceeds allowed length of 5 lines (6).

(maxbodylength)